jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp

```diff
@@ -38,6 +42,55 @@
  */
 
 /**
+ * Insert leading zeros to a string of a number if needed.
+ */
+static void
+ecma_date_insert_leading_zeros (ecma_string_t **str_p, /**< input/output string */
+                                ecma_number_t num, /**< input number */
+                                int32_t div) /**< "length" {10, 100, 1000} */
```

Comment:
Maybe, `leading_zeroes_num` would be better. In the case value of the argument should actually be number of leading zeroes to insert.